html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Fingerbank/Settings.pm

```diff
@@ -128,6 +129,17 @@ sub index :Path :Args(0) :AdminRole('FINGERBANK_READ') {
     $c->response->status($status);
 }
 
+sub update_p0f_map :Local :Args(0) :AdminRole('FINGERBANK_UPDATE') {
```

Comment:
The map is a few kilobytes so I'm not sure.

It adds weight to the code for a case that won't happen